app/services/draft_edition_updater.rb

```diff
@@ -10,7 +10,7 @@ def perform!
   def failure_reason
     if !edition.pre_publication?
       "A #{edition.state} edition may not be updated."
-    elsif edition.access_limited? && @options[:current_user].present? && edition.edition_organisations.map(&:organisation_id).exclude?(@options[:current_user].organisation.id)
+    elsif validates_access_limit_does_not_exclude_current_user? && access_limit_excludes_current_user?
```

Comment:
I'm a little confused reading this if statement - is this more like - "user is able to access limit" and "user does not belong to document organisations"?

Reply:
It's not quite that, that but I'll have a think to see if I can come up with some better names because the current ones aren't very clear.

Reply:
Let me know if you think the renamed methods are more expressive @minhngocd 🤞